src/ZODB/FileStorage/FileStorage.py

```diff
@@ -127,12 +127,12 @@ def __init__(self, afile):
         self._file = afile
 
 @implementer(
-        IStorage,
         IStorageRestoreable,
         IStorageIteration,
         IStorageUndoable,
         IStorageCurrentRecordIteration,
         IExternalGC,
+        IStorage,
```

Comment:
`IStorageRestoreable` is definitely an extension of `IStorage`, as is `IStorageUndoable`, so to me it seems redundant to leave `IStorage` in the list at all.

But then again, `IStorageIteration` is named similarly and *doesn't* extend `IStorage`, so people can disagree. I wonder what others think.

Reply:
I moved `IStorage` around until zope.interface IRO checking was happy, I am +0 with removing IStorage from the list.

Reply:
Sounds good to me!